src/Reflection/SignatureMap/Php8SignatureMapProvider.php

```diff
@@ -157,7 +157,7 @@ public function getFunctionSignature(string $functionName, ?string $className, i
 		}
 
 		if ($this->functionSignatureMapProvider->hasFunctionSignature($functionName, 1)) {
-			return $this->functionSignatureMapProvider->getFunctionSignature($functionName, $className, $variant);
+			return $this->functionSignatureMapProvider->getFunctionSignature($functionName, $className, 1);
 		}
```

Comment:
This seems like a bug to me. Previous we check `$variant > 0` and pass that variant value. 

```
		if ($variant > 0) {
			return $this->functionSignatureMapProvider->getFunctionSignature($functionName, $className, $variant);
		}
```

But I just realized this may have broken passing `$variant = 0`